samples/compose_app/src/main/java/org/telnyx/webrtc/compose_app/ui/screens/HomeScreen.kt

```diff
@@ -636,14 +637,24 @@ fun ConnectionState(state: Boolean) {
 }
 
 @Composable
-fun CurrentCallState(state: CallState) {
+fun CurrentCallState(state: TelnyxSocketEvent) {
     val callStateColor = when (state) {
-        CallState.RECONNECTING -> ReconnectingIconColor
-        CallState.DROPPED -> DroppedIconColor
-        CallState.RINGING -> RingingIconColor
+        is TelnyxSocketEvent.OnIncomingCall -> Color.Black
```

Comment:
I know incoming call is black on iOS but to me it maybe makes sense to use the neutral gray of (MaterialTheme.colorScheme.tertiary) so that it is consistent